nefertari/utils/data.py

```diff
@@ -94,3 +94,43 @@ def obj2dict(obj, classkey=None):
         return data
     else:
         return obj
+
+
+class FieldData(object):
```

Comment:
I'm not sure why we need this class. Seems like it groups info from the model along with info about field changes. But it's used in events where the model is already present. Maybe this class makes things a bit easier, but I'd be inclined to say that it's sufficient to place a dict mapping field names to new values directly on the event rather than creating a dicty that maps field names to instances of this class. Again, perhaps I haven't read far enough in the code the understand the purpose of this class yet.